library/calendar.po

```diff
@@ -119,24 +165,35 @@ msgid ""
 "Days returned will be tuples consisting of a year, a month, a day of the "
 "month, and a day of the week numbers."
 msgstr ""
+"Retorna um iterador para o mês month no ano year semelhante a :meth:"
+"`itermonthdates`, mas não restrito pelo intervalo de :class:`datetime.date`. "
+"Os dias retornados serão tuplas consistindo de um ano, um mês, um dia do "
+"mês e um número do dia da semana."
```

Comment:
Realmente não saquei muito como esse "numbers" no final se encaixa. Talvez seja algo como "em números", em oposição a "por extenso".

```suggestion
"Retorna um iterador para o mês *month* no ano *year* semelhante a :meth:"
"`itermonthdates`, mas não restrito pelo intervalo de :class:`datetime.date`. "
"Os dias retornados serão tuplas consistindo de números de um ano, um mês, um dia do "
"mês e um dia da semana."
```

Reply:
Esse `numbers`, a julgar por estar no plural, diz respeito a tudo que vem antes, não só ao `day of the week`. É uma coisa que em português não diriamos. Como não diriamos também aquele artigo antes dos itens da tupla. 

```python
>>> for m in calendar.Calendar().itermonthdays4(2024, 8):
...     print(m)
... 
(2024, 7, 29, 0)
(2024, 7, 30, 1)
(2024, 7, 31, 2)
(2024, 8, 1, 3)
(2024, 8, 2, 4)
(2024, 8, 3, 5)
(2024, 8, 4, 6)
(2024, 8, 5, 0)
(2024, 8, 6, 1)
(2024, 8, 7, 2)
(2024, 8, 8, 3)
(2024, 8, 9, 4)
(2024, 8, 10, 5)
(2024, 8, 11, 6)
(2024, 8, 12, 0)
(2024, 8, 13, 1)
(2024, 8, 14, 2)
(2024, 8, 15, 3)
(2024, 8, 16, 4)
(2024, 8, 17, 5)
(2024, 8, 18, 6)
(2024, 8, 19, 0)
(2024, 8, 20, 1)
(2024, 8, 21, 2)
(2024, 8, 22, 3)
(2024, 8, 23, 4)
(2024, 8, 24, 5)
(2024, 8, 25, 6)
(2024, 8, 26, 0)
(2024, 8, 27, 1)
(2024, 8, 28, 2)
(2024, 8, 29, 3)
(2024, 8, 30, 4)
(2024, 8, 31, 5)
(2024, 9, 1, 6)
```